src/components/SharedComponents/DateDisplay.js

```diff
@@ -8,16 +8,19 @@ import IconMaterial from "react-native-vector-icons/MaterialIcons";
 import { formatApiDatetime } from "sharedHelpers/dateAndTime";
 
 type Props = {
+  pretext?: string,
```

Comment:
"pretext" is also a noun, I was confused at first, but I see what you mean. Maybe "label"